File: jdroid-android-sample/src/main/java/com/jdroid/android/sample/ui/tablets/TabletActivity.java
package com.jdroid.android.sample.ui.tablets;

import androidx.fragment.app.Fragment;

import com.jdroid.android.activity.HorizontalFragmentsContainerActivity;

public class TabletActivity extends HorizontalFragmentsContainerActivity {

	@Override
	protected Class<? extends Fragment> getLeftFragmentClass() {
		return LeftTabletFragment.class;
	}

	@Override
	protected Class<? extends Fragment> getRightFragmentClass() {
		return RightTabletFragment.class;
	}
}

File: jdroid-android-sample/src/main/java/com/jdroid/android/sample/ui/tablets/TabletActivity.kt
package com.jdroid.android.sample.ui.tablets

import androidx.fragment.app.Fragment

import com.jdroid.android.activity.HorizontalFragmentsContainerActivity

class TabletActivity : HorizontalFragmentsContainerActivity() {

    override fun getLeftFragmentClass(): Class<out Fragment>? {
        return LeftTabletFragment::class.java
    }

    override fun getRightFragmentClass(): Class<out Fragment>? {
        return RightTabletFragment::class.java
    }
}
